tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -576,6 +583,21 @@ describe(__filename, () => {
     expect(root.find(FlagReviewMenu)).toHaveLength(0);
   });
 
+  it('renders the flag button when not signed in', () => {
```

Comment:
Actually, I just realized that you don't need this test because we already have one:

https://github.com/mozilla/addons-frontend/blob/61424ad5a9a200c5abfe3dfda013c1fd916217fa/tests/unit/amo/components/TestAddonReviewCard.js#L556

However, I would suggest changing the spec from 'lets you flag a review' to 'shows FlagReviewMenu when signed out'